Client/Frontend/Browser/SessionData.swift

```diff
@@ -67,4 +67,15 @@ class SessionData: NSObject, NSSecureCoding {
     static var supportsSecureCoding: Bool {
         return true
     }
+    
+    private static func updateSessionURLs(urls: [URL]) -> [URL] {
+        return urls.compactMap { url in
+            if PrivilegedRequest.isWebServerRequest(url: url),
```

Comment:
Maybe you can add convenience method inside  PrivilegedRequest for checking if it is Webserver request and privileged and return stripped url and use it also inside Tab class.